packages/preview/truthfy/0.1.0/typst.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "truthfy"
+version = "0.1.0"
+entrypoint = "main.typ"
+authors = ["Thumus"]
+license = "MIT"
+description = "Make truth table"
+repository = "https://github.com/ThumusLive/truthfy"
+homepage = "https://github.com/ThumusLive/truthfy"
+keywords = [
+    "truth",
+    "table",
+]
+exclude = []
+
+[utpm]
+version = "1"
```

Comment:
What is the purpose of this key? I couldn't find any explanation on https://github.com/ThumusLive/utpm. Why does this need to be part of the package manifest?

Reply:
I'm working on it, I'm gonna change things in the future so I need to track the version (but it's not published yet)
I'll delete it